src/app/components/members/Member.js

```diff
@@ -4,17 +4,25 @@ import { Col } from 'react-bootstrap'
 export default class Member extends Component {
   render () {
     const style = { backgroundImage: 'url(' + this.props.img + ')' }
-    const email = 'mailto:' + this.props.mail
-    const twitterLink = 'https://twitter.com/' + this.props.twitter
-    const gitHubLink = 'https://github.com/' + this.props.github
-
+    let mailLink
+    if (typeof this.props.mail !== 'undefined') {
+      mailLink = <a target='_blank' href={'mailto:' + this.props.mail} className='social-icon fa fa-envelope-o' />
+    }
+    let twitterLink
+    if (typeof this.props.twitter !== 'undefined') {
+      twitterLink = <a target='_blank' href={'https://twitter.com/' + this.props.twitter} className='social-icon fa fa-twitter' />
+    }
+    let gitHubLink
+    if (typeof this.props.github !== 'undefined') {
```

Comment:
When I talk about reliable, I mention the fact if `a` is not defined this will throw a ReferenceError:

![image](https://cloud.githubusercontent.com/assets/4710889/20539506/26f5788a-b0ed-11e6-9ee3-0d0d298ce392.png)